lib/elixir/lib/option_parser.ex

```diff
@@ -8,6 +8,10 @@ defmodule OptionParser do
   @type errors  :: [{String.t, String.t | nil}]
   @type options :: [switches: Keyword.t, strict: Keyword.t, aliases: Keyword.t]
 
+  defmodule InvalidOptionError do
```

Comment:
We're calling this `InvalidOptionError` but we're raising it with multiple errors for different options. Maybe we can call it `InvalidOptionsError`?